docs/team/zhtang29.md

```diff
@@ -5,37 +5,44 @@ title: Tang Zong Hao's Project Portfolio Page
 
 ### Project: RIZZipe
 
-RIZZipe is a desktop recipe book application used for busy cooks who want to track their recipes. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+RIZZipe is a desktop recipe book application used for busy cooks who want to track their recipes. The user interacts 
+with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
 
 Given below are my contributions to the project.
 
-* **FeatureName**: Added the ability to ....
-    * What it does: allows the user to...
-    * Justification: This feature improves...
-    * Highlights: This enhancement affects...
-    * Credits: *{if any}*
-
-* **New Feature**: Added...
-
 * **Code contributed**: [RepoSense link](https://nus-cs2103-ay2223s2.github.io/tp-dashboard/?search=zhtang29&breakdown=true)
 
+* **New Feature**: 
+* **Substitution** ([\#196]()) 
+  * What id does: Allows users to search for ingredient substitutions for any ingredient in their recipe
+  * Justification: Enables users to find replacements for missing ingredients that they may be lacking while cooking
+  * Highlights:
+    * Implemented a preloaded list of popular condiments with their common substitutes
+    * Substitution feature allows users to search up substitutions from those which they stored as well
+    
 * **Project management**:
+    * Managed documentation for release `v1.3` on GitHub
 
 * **Enhancements to existing features**:
-    * Updated... (Pull requests [\#](), [\#]())
-    * Wrote additional tests... (Pull requests [\#](), [\#]())
+    * Initial refactoring of `model` package ([\#65](), [\#77]())
+    * Wrote additional tests to ensure 100% coverage for `model` package ([\#260]())
 
 * **Documentation**:
   * User Guide:
-    * Added documentation for the feature `add` [\28](https://github.com/AY2223S2-CS2103T-T13-2/tp/pull/28)
+    * Added documentation for the features `add` [\28](), `edit` [\#179](), `addf` [\#179](), `sub`[#201]()
+    * Added import/export instructions [\#179]()
+    * Formatted and updated User Guide to reflect Keyboard Shortcuts
+    * Added Keyboard Shortcuts summary table
```

Comment:
add more contributions for UG leh. u pretty much tanked more than half the UG for v1.3